### PR TITLE
Rewrite of text formatting update, game def must now enable RichText …

### PR DESCRIPTION
**Pull request by brine**
…as a property type

### CODE REVIEW

**Comment by brine** [comment]
so I essentially reverted the previous update, then added a proper RichText custom property type to go alongside integer and string.  All the places in OCTGN that handle card properties will now treat RichText properties differently than regular string properties, so it should fix all the issues that cropped up for games that didn't use RichText properties in the first place.

I tested this branch with all the games that previously reported problems and they're all working fine here, as well as a few other potential issues, so it should be safe to deploy at this point.

File: octgnFX/Octgn.DataNew/Entities/Card.cs [hunk 1]
```diff
@@ -66,7 +66,7 @@ public object Clone()
                               Size = this.Size.Clone() as CardSize,
                               Properties =
                                   this.Properties.ToDictionary(
-                                      x => x.Key.Clone() as PropertyDef, x => x.Value.Clone() as PropertyDefValue)
+                                      x => x.Key.Clone() as PropertyDef, x => x.Value)
```
[the remark this answers is not in the document]
Comment (kellyelton) [reply octgnFX/Octgn.DataNew/Entities/Card.cs]: If that's not the case just let me know

[the remark this answers is not in the document]
Comment (brine) [reply octgnFX/Octgn.DataNew/Entities/Card.cs]: https://github.com/octgn/OCTGN/commit/02cc6a5700a70a4132dc8a0f99fdc046de22c97b#diff-d5e8801a4c39850e8c8195296c269688

This was actually just a reversion to the prior state, did we want to actually clone it?

[the remark this answers is not in the document]
Comment (kellyelton) [reply octgnFX/Octgn.DataNew/Entities/Card.cs]: Oh ok, I must be thinking of a different spot.

**Pull request merged**